kafka/producer/kafka.py

```diff
@@ -8,6 +8,8 @@
 import time
 import weakref
 
+from ..vendor import six
```

Comment:
I noticed the import style was inconsistent between `kafka/producer/kafka.py` and `kafka/consumer/group.py`, not sure why...?

Reply:
I think I must have started kafka/consumer/group.py before deciding to use relative imports. I like this style better.